setup-env

```diff
@@ -111,6 +125,24 @@ else
 fi
 set -o nounset
 
+# List Python versions and select one interactively
+if [ $LIST_VERSIONS -ne 0 ]; then
+  echo Available Python versions:
+  pyenv versions --bare --skip-aliases --skip-envs
+  read -r -p "Enter the desired Python version: " PYTHON_VERSION
+fi
+
+# Check if PYTHON_VERSION isn't empty. If it is installed, set it locally.
+if [ -n "$PYTHON_VERSION" ]; then
+  if pyenv versions --bare --skip-aliases --skip-envs | grep --fixed-strings "$PYTHON_VERSION" > /dev/null; then
```

Comment:
I prefer this syntax, but if you disagree please disregard:
```suggestion
  if pyenv versions --bare --skip-aliases --skip-envs | grep --fixed-strings "$PYTHON_VERSION" >/dev/null; then
```

Reply:
> I prefer this syntax, but if you disagree please disregard:

I was fine with how it was originally but linting changed it